src/extension/background-script/connectors/citadel.ts

```diff
@@ -79,6 +80,14 @@ class CitadelConnector implements Connector {
     );
   }
 
+  // @Todo: implement function call
+  async getTransactions(): Promise<GetTransactionsResponse> {
+    console.error(
```

Comment:
I think we should not implement all these functions in a single PR.

Maybe we could just do a console warning and return an empty list for the other connectors (their functionality can stay unchanged )?

Reply:
... and already merge into master? How would it work until finishing it, one connector uses the new logic of fetching transactions from the connector all of the rest fetch from IndexDB as currently is?

Reply:
Yes. Or we change the merging strategy (my other comment)

Reply:
In the community call we decided to try to implement all in one PR for now, so I'll resolve this conversation.